meta-balena-common/recipes-core/initrdscripts/initramfs-framework_%.bbappend

```diff
@@ -11,7 +12,10 @@ SRC_URI_append = " \
     "
 
 do_install_append() {
-    install -m 0755 ${WORKDIR}/prepare ${D}/init.d/70-prepare
+    install -m 0755 ${WORKDIR}/prepare ${D}/init.d/02-prepare
+    install -m 0755 ${WORKDIR}/fsuuidsinit ${D}/init.d/03-fsuuidsinit
+    rm ${D}/init.d/01-udev
```

Comment:
Not sure I understand the need for removing this and then reinstalling in 04-udev

Reply:
I think prepare can be run first by marking it `001` or`0000` or something like that. there is a null workaround using something similar.

Reply:
So, the order is:

- debug
- console_null
- prepare
- fsuuidinit
- udev

Currently udev is 01-udev, so we would need to have:
- 00-debug
- 000-console_null
- 0000-prepare
- 00000- fsuuid
- 01-udev

So instead of all those zeros I think having a 04-udev makes more sense.

Reply:
As per the latest arch call I will be moving the udev module up again to cater for the regenerating UUIDs also on HUP. This will address this concern.